docker-compose/docker-compose.yaml

```diff
@@ -52,10 +52,13 @@ services:
       dockerfile: waltid-services/waltid-verifier-api/Dockerfile
     depends_on:
       - caddy
+      - opa-server
```

Comment:
I understand the dependency, but perhaps there's a way to make it conditionally, based on profile or anything else
because currently, as a user, I am forced to use opa server, although I might not need dynamic policies for my use case